### DISCUSSION TITLE
Defect Report

### DISCUSSION THREAD

**Discussion by nd-hsnaa** [kimai/kimai#3376] [General]
Greetings, me and my fellow team members from Software Engineering department of Science Computer faculty in University of Malaya had conducted a software testing for group assignment. There are few defects that we had found on the list below. Hope it helps the future development of Kimai! 

1. Users cannot save tasks after trying to record expenses. - When a category is chosen, save button will not work, Project and Activity will disappear. Invalid data error would also be displayed.

2. User cannot search record using a specific term.- When user input a search term, only error message will be displayed. However, the term actually exist. 

3. Role management - The new role is not added automatically into the table. Need to manually refresh the page to see the new user role.

**Comment by kevinpapst** [comment]
Hi & thanks for your feedback!

1. I don't understand that one. Can you share a video? Which user (role) are you using to see this problem?
2. Which screen? User timesheets or team timesheets or user administration? In which field is the term saved? Please share more details.
3. Will be solved with https://github.com/kevinpapst/kimai2/pull/3377
